app/blueprints/assessments/templates/macros/comments.html

```diff
@@ -19,19 +19,26 @@ <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Comments</h2>
             <div class="comment-group">
                 <p class="govuk-body">{{ comment.updates[-1]['comment'] }}</p>
                 <p class="govuk-body-s">{{ comment.full_name }} ({{ comment.highest_role|all_caps_to_human }}) {{ comment.email_address }}</p>
-                <p class="govuk-body-s">{{ comment.date_created|utc_to_bst }}</p>
+                <p class="govuk-body-s">{{ comment.updates[-1].date_created|utc_to_bst }}</p>
                 {% if g.account_id == comment.user_id %}
                     <div class="govuk-button-group">
                         <a class="govuk-link" href="{{ url_for(
                             "assessment_bp.display_sub_criteria",
                             application_id=comment.application_id,
                             sub_criteria_id=comment.sub_criteria_id,
+                            theme_id=comment.theme_id,
                             edit_comment="1",
                             comment_id=comment.id
                         ) }}" >Edit comment</a>
-                        {# {% if comment.updates|length > 1 %}
-                            <a class="govuk-link" href="./comment-history.html">See history</a>
-                        {% endif %} #}
+                        {% if comment.updates|length > 1 %}
```

Comment:
Does the logged in user need to be the commenter in order to view it's history

Reply:
`See History` option is available for all the roles (`Commenter`, `Assessors` & `Lead Assessors`). So logged in user must have at least one role. I think `Commenter` role is assigned by default to any assessor